lib/OpenQA/WebAPI/Plugin/ObsRsync/Controller/Folders.pm

```diff
@@ -196,4 +197,29 @@ sub forget_run_last {
     return $self->render(json => {message => "error $!"}, status => 500);
 }
 
+sub latest_test {
+    my $self  = shift;
+    my $alias = $self->param('alias');
+    my $full  = $self->param('full');
+    my $id    = $self->obs_rsync->get_last_test_id($alias);
+    return undef unless $id;
+
+    return $self->render(json => {id => $id}, status => 200) unless $full;
+    my $result = '';
+    try {
+        my $job = $self->schema->resultset("Jobs")->search(
+            {
+                id => $id
+            })->first;
```

Comment:
can be shortened to `my $job = $self->schema->resultset("Jobs")->single({id => $id});